packages/fluentui/docs-components/src/knobs/defaultComponents.tsx

```diff
@@ -47,7 +47,7 @@ const KnobNumber: React.FunctionComponent<KnobNumberKnobComponentProps> = props
       const newValue = parseInt(e.target.value, 10);
       const min = parseInt(props.min, 10);
       const max = parseInt(props.max, 10);
-      props.setValue(newValue < min ? props.min : newValue > max ? props.max : newValue);
+      props.setValue(newValue < min ? props.min : newValue > max ? props.max : newValue || props.min);
```

Comment:
_nit_
Please could you do that in a bit more readable way, and not ternary inside ternary and or to finish it all? :)

Reply:
The alternative would be something like:

```javascript=
let newValue = parseInt(e.target.value, 10);
if(newValue < min){
    newValue = props.min;
} else if(newValue > max){
    new value = props.max;
}

props.setValue(newValue || props.min)
```

Would that be better?

Reply:
LGTM!
@layershifter , are you OK with that change?

Reply:
Done